### PR TITLE
Add redirect from index.html to latest

### PR DESCRIPTION
**Pull request by kylefraser**
This PR adds a redirect to `index.html` to point to the index page of `latest`.

Uses `meta` tag as well as fallback `script`. 

Jira Ticket: [WEB-110](https://starburstdata.atlassian.net/browse/WEB-110)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-  <li><a href="./423-e/index.html">423-e</a></li>
-  <li><a href="./413-e/index.html">413-e</a></li>
-</ul>
-</body>
+    <ul>
+      <li><a href="./435-e/index.html">435-e</a> (latest)</li>
+      <li><a href="./429-e/index.html">429-e</a></li>
+      <li><a href="./423-e/index.html">423-e</a></li>
+      <li><a href="./413-e/index.html">413-e</a></li>
+    </ul>
```
Comment (jhlodin) [inline index.html]: Should remove these contents of the page with a blanket "redirecting to latest documentation" message. No need to have a list of versions that would either go out of date or need to be updated for no reason.

**Review by jhlodin — APPROVED** [review]
One suggested change to page contents, otherwise LGTM

File: index.html (hunk 1 above)
Comment (jhlodin) [reply index.html]: Probably with a static "Click here if you are not redirected" as a catchall.

**Review by mosabua — APPROVED** [review]
Looks good to me from functionality perspective. Adding a sentence with a link as suggested by Joe is probably a good idea.

**Comment by kylefraser** [comment]
@jhlodin Feedback is in. Likely good to merge whenever you'd like 👍

**Pull request merged**